Source/Core/UICommon/ResourcePack/ResourcePack.cpp

```diff
@@ -19,7 +20,7 @@
 
 namespace ResourcePack
 {
-constexpr char TEXTURE_PATH[] = "Load/Textures/";
+constexpr char TEXTURE_PATH[] = HIRES_TEXTURES_DIR DIR_SEP;
```

Comment:
I'm sorry, I missed this change at first. Looks like this changes Load/Textures/ to Textures/, is that intended?

Reply:
Yes, because the Load directory is configurable.

Reply:
Doh, you are right.